.github/workflows/StagedUpload.yml

```
@@ -32,14 +29,22 @@ jobs:
      - name: Download from staging bucket
        shell: bash
        env:
+         AWS_ENDPOINT_URL: ${{ secrets.S3_DUCKDB_STAGING_ENDPOINT }}
          AWS_ACCESS_KEY_ID: ${{ secrets.S3_DUCKDB_STAGING_ID }}
          AWS_SECRET_ACCESS_KEY: ${{ secrets.S3_DUCKDB_STAGING_KEY }}
        run: |
           TARGET=$(git log -1 --format=%h)
           mkdir to_be_uploaded
-          aws s3 cp --recursive "s3://duckdb-staging/$TARGET/${{ inputs.target_git_describe }}/$GITHUB_REPOSITORY/github_release" to_be_uploaded --region us-east-2
+          aws s3 cp --recursive "s3://duckdb-staging/$TARGET/${{ inputs.target_git_describe }}/$GITHUB_REPOSITORY/github_release" to_be_uploaded
 
      - name: Deploy
+        if: ${{ inputs.target_git_describe != '' }}
        shell: bash
+       env:
```

```suggestion
       if: ${{ inputs.target_git_describe != '' }}
       shell: bash
       env:
```
Maybe this? sorry for the noise